README.md

```diff
@@ -118,45 +160,99 @@ This library has a `peerDependencies` listing for `react-dom`.
 You may also be interested in installing `jest-dom` so you can use
 [the custom jest matchers](https://github.com/gnapse/jest-dom#readme)
 
-## Usage
+## Setup
+
+### Globals
+
+There are several options you can add to your global test config that simplify
+the setup and teardown of tests in individual files. For example, you can ensure
+[`cleanup`](#cleanup) is called after each test and import additional
+assertions.
+
+To do this with Jest, you can add the
+[`setupTestFrameworkScriptFile`](https://facebook.github.io/jest/docs/en/configuration.html#setuptestframeworkscriptfile-string)
+option to your Jest config. The setup file can be anywhere, but if you're using
+create-react-app, use the default `src/setupTests.js`.
```

Comment:
Change to:

```
... use the default `src/setupTests.js` (in this case, no configuration changes are necessary).
```

Reply:
Not sure I understand "no configuration changes are necessary". Can you clarify?

Reply:
Oh I see. It doesn't require changing the Jest config. I think it can be worded better than that even:

>>>
To do this with Jest, you can add the
`setupTestFrameworkScriptFile` option to your Jest config.

> Note:
> If you are using the default setup from `create-react-app`, this option is set to `src/setupTests.js`. You should create this file if it doesn't exist and put the setup code there.